Makefile

```diff
@@ -0,0 +1,234 @@
+# Makefile for Open Targets data pipeline
+# Based on steps in https://github.com/opentargets/data_release/wiki/OT011-Data-Processing-(Backend)
+
+# THhese variables can be overridden on the command-line
+ES_PREFIX = gp-makefile
+ELASTICSEARCH_NODES = http://localhost:9200
```

Comment:
does it @Docproc have to get the `es URL` from `db.ini` file better than having two different places to change it?

Reply:
I thought of that, but I don't think it's possible to easily get the value from the `db.ini` file into the Makefile, at least not without introducing additional dependencies (i.e. some sort of INI file parser). Is there an easy way I'm missing?

Reply:
We can also allow preset environment variables to override the ones inside the Makefile with `?=` syntax instead fo `=`as described https://stackoverflow.com/a/40241012

Reply:
I didn't know you could do that! Change made, see 0833d3f